bin/fresh

```diff
@@ -86,13 +86,30 @@ fresh_after_build() {
 }
 
 _fresh_preamble() {
+
+  local no_path_export fresh_path user_shell="$(getent passwd $LOGNAME | cut -d: -f7)"
```

Comment:
`getent` is not available on Mac OS X. Would it be possible to detect `fish` via the presence of an environment variable?